pkgs/unified_analytics/lib/src/analytics.dart

```diff
@@ -38,12 +38,18 @@ abstract class Analytics {
   /// An optional parameter [clientIde] is also available for dart and flutter
   /// tooling that are running from IDEs can be resolved. Such as "VSCode"
   /// running the flutter-tool.
+  ///
+  /// [enabledFeatures] is also an optional field that can be added to collect
+  /// any features that are enabled for a user. For example,
+  /// "enable-linux-desktop,cli-animations" are two features that can be enabled
+  /// for the flutter-tool.
   factory Analytics({
     required DashTool tool,
     required String dartVersion,
     String? flutterChannel,
     String? flutterVersion,
     String? clientIde,
+    String? enabledFeatures,
```

Comment:
So are all the tool-specific (i.e. Flutter tool-specific, analyzer-specific, etc) fields going to be optional constructor parameters on the public `Analytics` class?